lib/server.go

```diff
@@ -2102,14 +2102,12 @@ func (srv *Server) _handleBlock(pp *Peer, blk *MsgDeSoBlock) {
 		return
 	}
 
-	if pp != nil {
```

Comment:
Why don't we need this `pp != nil` check anymore?

Reply:
The peer can't be nil. Other parts of this code and the FastHotStuffConsensus.HandleBlock function would break if it was nil